packages/components/page-header/src/page-header.ts

```diff
@@ -1,14 +1,14 @@
-import { buildProps, iconPropType } from '@element-plus/utils'
+import { buildProps, definePropType } from '@element-plus/utils'
 import { Back } from '@element-plus/icons-vue'
-import type { ExtractPropTypes } from 'vue'
+import type { Component, ExtractPropTypes } from 'vue'
 import type PageHeader from './page-header.vue'
 
 export const pageHeaderProps = buildProps({
   /**
    * @description icon component of page header
    */
   icon: {
-    type: iconPropType,
+    type: definePropType<string | Component | null>([String, Object, Function]),
```

Comment:
Maybe you should add a `withNullIconPropType` in the same file as the `iconPropType` type.

Reply:
done.